SECURITY.md

```diff
@@ -0,0 +1,63 @@
+# Security Policy
```

Comment:
Should we link to anything official regarding what this is? Perhaps https://csrc.nist.gov/glossary/term/security_policy